next.dynamic.mjs

```diff
@@ -212,6 +212,21 @@ const getDynamicRouter = async () => {
       path
     );
 
+    const match = path.match(/blog\/(release|vulnerability)/);
```

Comment:
Should this have a `^` at the start to ensure some weird nested path on the site elsewhere never matches it, and perhaps a `(\/|$)` at the end, so that if a path like `/blog/release-something` ever existed it wouldn't accidentally match?

Reply:
that's a good idea I believe just to make sure, I will update the regex

Reply:
Done